setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = cloud-utils
-version = 0.0.1
+version = 2
```

Comment:
hmm, why the change here?

Reply:
I would like to specify the package version when importing it in requirements.txt, as we do with gamla.
This way when inserting changes such as function name modified we can update the version of the package in the projects that use it, and have control over the version.

Reply:
Actually I'm seeing that we use the commit hash as version, so I will keep the same version number.